### PR TITLE
[APP-720] Paginate custom feeds

### PR DESCRIPTION
**Pull request by ansh**
Paginate "Discover Feeds" screen

### CODE REVIEW

File: src/state/models/discovery/feeds.ts [hunk 1]
```diff
@@ -89,9 +113,24 @@ export class FeedsDiscoveryModel {
...
+    // 1. set feeds data to empty array
     this.feeds = []
+
+    // 2. push data into feeds array
+    for (const f of res.data.feeds) {
+      this.feeds.push(new CustomFeedModel(this.rootStore, f))
+    }
+
+    // 3. set loadMoreCursor
+    this.loadMoreCursor = res.data.cursor
```
Comment (estrattonbailey) [inline src/state/models/discovery/feeds.ts:125]: Nit: could dry this up by resetting `this.feeds = []` and then calling `this._append(res)`, but fine with me if you wanna keep things separate and simple too 👍

Comment (estrattonbailey) [inline src/state/models/discovery/feeds.ts:125]: Nit: could dry this up by resetting this.feeds = [] and then calling this._append(res), but fine with me if you wanna keep things separate and simple too 👍

File: src/state/models/discovery/feeds.ts [hunk 2]
```diff
@@ -48,16 +54,34 @@ export class FeedsDiscoveryModel {
...
     } catch (e: any) {
       this._xIdle(e)
     }
   })
 
+  loadMore = bundleAsync(async () => {
+    this._xLoading()
+    if (!this.hasMore) {
+      return
+    }
```
Comment (estrattonbailey) [inline src/state/models/discovery/feeds.ts:71]: Should the `this._xLoading()` call happen after the `this.hasMore` check?

Comment (ansh) [reply src/state/models/discovery/feeds.ts:71]: You're right. Good catch!

File: src/state/models/discovery/feeds.ts (hunk 1 above)
Comment (ansh) [reply src/state/models/discovery/feeds.ts:125]: Yeah true. That is the convention in other model files so I can follow it.

**Review by estrattonbailey — APPROVED** [review]
LGTM!

**Review by pfrazee — APPROVED** [review]
LGTM!

**Pull request merged**